### DISCUSSION TITLE
Picture in picture as in "Telegram"

### DISCUSSION THREAD

**Discussion by bropines** [ReVanced/revanced-discussions#280] [Question]
I would like to see a patch that makes a more beautiful PIP. Smoother and more beautiful.

**Comment by oSumAtrIX** [comment]
PiP is an Android feature. You have to submit your issue at Android not ReVanced

**Reply by bropines** [reply]
> PiP is an Android feature. You have to submit your issue at Android not ReVanced

At the same time, PIP in Telegram works on the 9, 10, 11 versions of android just as beautifully and smoothly. And at the same time it has rounded edges.

**Comment by bropines** [comment]
![Screenshot_2022-07-02-18-45-20-469_com miui home](https://user-images.githubusercontent.com/57861007/177003615-f8be029e-2373-4cf7-8ada-3230c4202cb3.jpg)
![Screenshot_2022-07-02-18-44-41-231_com miui home](https://user-images.githubusercontent.com/57861007/177003618-130cc6c1-c38f-4b5e-bd83-edece9f9e432.jpg)

**Reply by bogadana** [reply]
what is this supposed to show

**Reply by bropines** [reply]
> what is this supposed to show

Well, look, Telegram somehow implemented this even where it seems like it's impossible. And what is worse than ReVanced?

**Reply by bogadana** [reply]
this is exactly how youtube looks for me. theres something wrong with your os

**Reply by bropines** [reply]
> this is exactly how youtube looks for me. theres something wrong with your os

well, for example, there is no such type on Android below 12. Or you have a custom ROM
